pkg/util/ansible_execution.go

```diff
@@ -124,6 +124,10 @@ func AnsibleExecution(
 		}
 		ansibleEE.Spec.ExtraVars["edpm_service_name"] = json.RawMessage([]byte(fmt.Sprintf("\"%s\"", service.Name)))
 
+		if len(deployment.Spec.ServicesOverride) > 0 {
+			ansibleEE.Spec.ExtraVars["edpm_services_override"] = json.RawMessage([]byte(fmt.Sprintf("\"%s\"", deployment.Spec.ServicesOverride)))
+		}
+
```

Comment:
whenever a variable is set in ansibleEE.Spec.ExtraVars by our code, we should probably first check if it isn't already set, so that the code does not overwrite something set by the user in Deployment.Spec.AnsibleExtraVars which gets passed through as ansibleEE.Spec.ExtraVars.

i'd be fine to merge this as-is though, and we can discuss if that is something we want to check for and apply it to the earlier statements above as well.

Reply:
In principle we should trust that our customer, if they define the var, knows what they are doing. But I agree that it's not a deal breaker.